site/viamguides/src/cv-checkout/cv-checkout.md

```diff
@@ -0,0 +1,257 @@
+1. Paste the copied `video_path` value into your camera component's `video_path` input, which is in the **Attributes** section:
+![paste video_path value](assets/pasteVideoPath.png)
+1. Click **Save** in the top right once more to save and apply your configuration changes.
+1. Expand your camera component's **TEST** panel. If things are properly configured, you should see the image streaming from your camera.
+   ![test camera feed](assets/testCamera.png)
+1. With your camera added and working, you can now delete the `discovery-1` service and `find-webcams` module as you'll no longer need them. Click the *...* next to each item, then select **Delete**. 
+   <br>
+   <img src="assets/deleteComponent.png" alt="delete component" width=400 />
+1. You will be prompted to confirm the deletion, select **Delete**
+   <br>
+   <img src="assets/confirmDelete.png" alt="confirm component deletion" width=350 />
+1. Finally, **Save** your configuration changes.
+
+Great, your machine now has eyes! 
+
+<!-- ------------------------ -->
+
+## Create a dataset and capture some training data
+Duration: 15
+
+To train a custom model based on your beverages, you'll need a dataset the TensorFlow Lite framework can use. Here, you'll create a dataset and add some images of your beverages using the webcam you configured in the last step.
+
+1. In [the Viam app](https://app.viam.com/data/), find the **DATASETS** tab.
+1. Click the **+ Create dataset** button and give your dataset a name, like `beverages`. Click the **Create dataset** button again to save.
+   ![create a dataset](assets/createDataset.png)
+1. Switch back to your machine in [the Viam app](https://app.viam.com/fleet/locations). You can navigate back by going to Fleet > All Machines Dashboard, then clicking on the name of your machine.
+1. Expand your camera component's **TEST** panel. Here, you'll see the live feed of your camera as well the "Add image to dataset" icon, which looks like a camera.
+   ![add image to dataset button](assets/addImageButton.png)
+1. Using the live feed as a viewfinder, position your webcam so that you can place one of your beverages fully in the frame. The less visual clutter in the background the better!
+   ![positioning a beverage for capture](assets/positionDrink.jpeg)
+   > aside positive
+   > **Good test data**: If you're able to set up a little "photo shoot" booth to capture images of your beverages, do so! My own setup consisted of a piece of cardboard propped up by a bookend, my laptop pushed close enough to let the cardboard fill the frame, and enough space to place my drink. Great images are clear, well-lit, minimize visual noise or clutter, and clearly isolate the object you are trying to capture. If possible, capturing different angles and different lighting scenarios are great additions to your dataset as well. Remember, for each object you are trying to detect, at least 10 images are needed to train a model!
```

Comment:
This is a good tip. Is it accurate to recommend users to collect data from a similar condition as to what they'll be using their model in? e.g. Bright outdoor setting Or is it more important that it just be a controlled and similar background.

Reply:
I think it's more important to have as controlled and similar background as possible. Even in the codelab, I took photos of my drinks in a "photo shoot"-like setup, but tried the vision service with myself in the frame, and the model did decently. :)